api/v1alpha1/openapi.yaml

```diff
@@ -2115,11 +2115,10 @@ components:
         containers:
           type: object
           properties:
-            matchPattern:
+            matchPatterns:
               type: array
               items:
                 type: string
-                additionalProperties: true
```

Comment:
what does additionalProperties mean?

Reply:
Right I was looking at the code generation now, and it didn't change... so... good cleanup. Thanks @avishayt